src/Microsoft.AspNet.Authentication.OpenIdConnect/OpenIdConnectHandler.cs

```diff
@@ -669,7 +669,6 @@ public OpenIdConnectHandler(HttpClient backchannel)
             var responseMessage = await Backchannel.SendAsync(requestMessage);
             responseMessage.EnsureSuccessStatusCode();
             var userInfoResponse = await responseMessage.Content.ReadAsStringAsync();
-            var userInfoEndpointJwt = new JwtSecurityToken(userInfoResponse);
             var user = JObject.Parse(userInfoResponse);
```

Comment:
No, we want UserInfoReceivedContext to take a JObject so the developer doesn't have to deal with the same problem. I have a fix but I don't have a good way of testing it. Are there any IDPs we use that support JWT responses here?